src/Styleguide/Pages/Artist/Routes/Articles/index.tsx

```diff
@@ -1,8 +1,12 @@
 import React from "react"
-import { ArticlesQueryRenderer } from "./ArticlesQueryRenderer"
+import { ArticlesRefetchContainer } from "./ArticlesRefetchContainer"
```

Comment:
Is this still in a separate file because of how it was but is it moving into a single file?

Reply:
Yup, i'm gradually moving @mzikherman's work over into the new structure. This import can live flat in this file.